docs/UserGuide.md

```diff
@@ -174,8 +174,25 @@ _Details coming soon ..._
 
 ## FAQ
 
-**Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
+### **Q**: How do I transfer my data to another Computer?<br>
+ - #### **A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous MediBase3 home folder.<br>
+
+### **Q**: How do we open the command terminal?<br>
+ - #### **A**: Windows - Press `Win + R`, type `cmd` and press `Enter`.<br>
+ - #### **A**: Mac - Press `Cmd + Space`, type `terminal` and press `Enter`.<br>
+ - #### **A**: Linux - Press `Ctrl + Alt + T` to open the command terminal.<br>
+
+### **Q**: How to install Java 17<br>
+ - #### **A**: Download the Java 17 [here](https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html). Then follow the [installation guide](https://docs.oracle.com/en/java/javase/17/install/overview-jdk-installation.html).<br>
+
+### **Q**: How to know if you have Java 17<br>
+ - #### **A**: Open your command terminal and enter `java --version`. The first line should display `java` followed by the `version number`.<br>
+
+### **Q**: Is there a limit to the number of patients/appointments we can add?<br>
+ - #### **A**: As of right now, We do not have a limit to the number of patients/appointments but is dependent on the hardware specification.<br>
+
+### **Q**: Do you support non-English inputs?<br>
+ - #### **A**: We do not support non-English inputs, but we are working on it!<br>
```

Comment:
Hm do you want to "####" the answer lines also, cos it will show up in the table of content and be orange like the question lines

Reply:
Yea, I think it would be better to only "###" the question lines

Reply:
Resolved